framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -252,16 +252,13 @@ public StructuralEqualityComparer createEqualityComparer() {
                     @Override
                     protected boolean arePrimeAnnosEqual(
                             AnnotatedTypeMirror type1, AnnotatedTypeMirror type2) {
-                        type1.replaceAnnotation(
-                                convertSpecialIntRangeToStandardIntRange(
-                                        type1.getAnnotationInHierarchy(UNKNOWNVAL)));
-                        type2.replaceAnnotation(
-                                convertSpecialIntRangeToStandardIntRange(
-                                        type2.getAnnotationInHierarchy(UNKNOWNVAL)));
-                        type1.replaceAnnotation(
-                                convertToUnknown(type1.getAnnotationInHierarchy(UNKNOWNVAL)));
-                        type2.replaceAnnotation(
-                                convertToUnknown(type2.getAnnotationInHierarchy(UNKNOWNVAL)));
+                        AnnotationMirror t1am = type1.getAnnotationInHierarchy(UNKNOWNVAL);
```

Comment:
This is a left-over from an alternative fix. It makes the code a bit easier to read, but I can remove this again.

Reply:
Does the TODO on line 258 still apply?

Reply:
Oh, I was looking at an intermediate commit.  I would prefer this change just be pushed to master, since it is not related to this pull request.